base.py

```diff
@@ -977,25 +977,22 @@ def get_python_ver():
         return [sys.version_info.major, sys.version_info.minor, sys.version_info.micro]
 
     @staticmethod
-    def update_webgpu_cts_expectations(project_root_dir, gpu_device_id):
+    def update_webgpu_cts_expectations(project_root_dir):
         expectation_file = f'{project_root_dir}/third_party/dawn/webgpu-cts/expectations.txt'
 
-        generation = Util.get_intel_gpu_generation(f'0x{gpu_device_id}')
-        if generation is None:
-            Util.warning(f'Failed to update {expectation_file} because of unknown generation')
-            return
-
-        old_gpu_tag = 'intel-gen-9'
-        new_gpu_tag = f'intel-gen-{generation}'
-        if old_gpu_tag == new_gpu_tag:
-            return
-
         if not os.path.exists(expectation_file):
             Util.warning(f'{expectation_file} does not exist')
             return
 
+        # Update all gpu tag 'intel-*', such as 'intel-gen-9' and 'intel-0x9bc5' to 'intel'
+        existing_gpu_tag_pattern = r'intel-[^\s]*'
+
+        # Hold temporary lines for duplication check, because there may be some expectations with same case, but different device id.
```

Comment:
```suggestion
        # Hold temporary lines for duplication check, because there may be some expectations with same case, but different device ids.
```